### DISCUSSION TITLE
How update parameters

### DISCUSSION THREAD

**Discussion by devscheffer** [dbeaver/dbeaver#12419] [Q&A]
once a set parameter like 
```sql
@set total = 12
```
it doesn't change the value total if I try to change the value from 12 to 15.

How can I change that? or is it a bug?

**Comment by LonwoLonwo** [comment]
![2024-01-04 15_48_43-DBeaver 23 3 1 - _postgres 6 1_ Script-122](https://github.com/dbeaver/dbeaver/assets/45152336/e1e354bf-e7ba-4678-8dde-c69fe018548a)

You can use the variables panel for it in the SQL Editor. 
(This button is configurable in the preferences -> toolbar customization.
